backend/btrixcloud/users.py

```diff
@@ -434,6 +434,11 @@ async def forgot_password(
             user.email, token, request and request.headers
         )
 
+        # Lock user until password is reset
```

Comment:
Hm, should this be here? That means if you also manually requested a password reset, but then remembered it later, your account is now locked - do we want to do that? I think should still be able to use account, if you remember password later and choose not to reset it.

Reply:
Removed